internal/benchrunner/runners/stream/runner.go

```diff
@@ -295,25 +341,25 @@ func (r *runner) collectGenerators() error {
 	for scenarioName, scenario := range r.scenarios {
 		config, err := r.getGeneratorConfig(scenario)
 		if err != nil {
-			return err
+			return fmt.Errorf("[%s]: %w", scenarioName, err)
```

Comment:
Thanks for adding more context here, I would also give context about the action that has failed here, something like this:
```suggestion
			return fmt.Errorf("failed to obtain generator configuration for scenario %q: %w", scenarioName, err)
```